.github/workflows/validate-workflow.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x]
+        node-version: [14.x]
```

Comment:
@bc-chaz and I were discussing this yesterday in standup. I believe the consensus was that if we want to enforce node v14, we'll have to go and update dev documentation alongside of it. We might have to hold on this change for now until we determine what we want to do.

Reply:
unfortunately, if the build is failing, we might have to update this bit. I'm less concerned about updating the workflow - since this only affects the checks on GH, unlike nvmrc or potentially engines.

all that said, we should hold off on merging this PR till we talk to dev docs and figure out a good path forward. I'm going to add a `do not merge` label to this PR.